veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,258 @@
+# VEP #21: Seamless TCP migration and vhost-user support with passt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, linking to VEP directory in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+KubeVirt users can benefit from seamless live migration of VMs while preserving TCP connections.
+This functionality has recently been introduced in `passt` together with `vhost-user` support, which increases network throughput. 
+This proposal outlines the integration plan to be implemented in KubeVirt. 
+
+## Motivation
+
+Currently, KubeVirt does not support all Primary Network requirements expected by users:
+- Seamless network connectivity on migration.
+- Observability tools such as [Stackrox Collector](https://github.com/stackrox/collector).
+- Service Mesh technologies such as [Istio](https://istio.io) and [Linkerd](https://linkerd.io/).
+- High throughput (on par with tap)
+
+Bridge binding does not support observability or service meshes, Masquerade hides the real IP from the user, and passt until recently lacked seamless migration and tap-level performance.
```

Comment:
Using bridge binding with the pod network makes the VM non-migratable.

https://github.com/kubevirt/kubevirt/blob/f8aa1f85ffca1a6a477bbb1d369097b3b0e36088/pkg/network/vmispec/interface.go#L57

*Unless some hack is being used.